packages/fast-development-site-react/app/components/button/button.tsx

```diff
@@ -1,20 +1,38 @@
 import * as React from "react";
+import manageJss, { IJSSManagerProps } from "@microsoft/fast-jss-manager-react";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
+import { IManagedClasses } from "@microsoft/fast-components-class-name-contracts-base";
+import { IDesignSystem } from "../../design-system";
 
 export interface IButtonProps {
     text: string;
     object: any;
 }
 
-class Button extends React.Component<IButtonProps, {}> {
+export interface ITestButtonClassNameContract {
+    button: string;
+}
+
+const ButtonStyles: ComponentStyles<ITestButtonClassNameContract, IDesignSystem> = {
+    button: {
+        width: "50px",
```

Comment:
`toPx()`?

Reply:
not necessary to import the utility library here since these are just test components to verify the functionality of the dev site